### DISCUSSION TITLE
How can I set the time zone for a linked source to Azure SQL Database?

### DISCUSSION THREAD

**Discussion by markallisongit** [Budibase/budibase#11508] [Help] [answered]
I have a table in Azure SQL Database called Flights. There are two columns in there of datatype `time` and I have linked this table to budibase. When I view the table in budibase the time has been offset by one hour. How can I stop it doing this?

Here's my budibase window

![image](https://github.com/Budibase/budibase/assets/20219805/93279a25-d6cf-4e7f-990e-a62899556ff1)

Here's the data in my Azure SQL Database

![image](https://github.com/Budibase/budibase/assets/20219805/484a1f4c-981a-4e2f-abe6-83bf6f78ae1c)

I tried editing the column, but there's no `Ignore TIme Zone` option.

![image](https://github.com/Budibase/budibase/assets/20219805/e0b41571-8f25-4dcf-b734-6a499210f6c9)

**Comment by melohagan** [comment, marked as answer]
Hey @markallisongit 

For some of the other SQL datasources, the solution is to toggle the *Ignore timezone* flag, but unfortunately that is not available for SQL Server.

You can read more about it here: https://github.com/Budibase/budibase/issues/10278#issuecomment-1516097194
